src/handlers/MessageCommandHandler.ts

```diff
@@ -152,6 +152,7 @@ export async function MessageCommandHandler(
 	}
 
 	if (!command.type.includes(CommandType.MESSAGE)) return;
+	if (!message.guild && command.dmPermission === false) return;
```

Comment:
Should this not also include general permissions for members?

Reply:
Yes, I'll do it